src/components/Card/Card.tsx

```diff
@@ -1,34 +1,41 @@
-import React, { ReactElement } from 'react'
-import CardBase, { CardBaseProps } from './components/CardBase'
-import CardFigure, { CardFigureProps } from './components/CardFigure'
-import CardNews, { CardNewsProps } from './components/CardNews'
-import CardVideo, { CardVideoProps } from './components/CardVideo'
+import React from 'react'
 
 export interface CardProps {
-    type: 'base' | 'news' | 'video' | 'figure'
+    link: string
+    header: string
+    description?: string
+    image: CardImageProps
 }
 
 export interface CardImageProps {
     src: string
     alt?: string
 }
 
-export interface TypeProps {
-    [index: string]: ReactElement
-}
-
-const Card: React.FC<
-    CardProps & CardBaseProps & CardNewsProps & CardFigureProps & CardVideoProps
-> = (props): JSX.Element => {
-    const { type = 'base' } = props
-
-    const cardTypes: TypeProps = {
-        base: <CardBase {...props} />,
-        news: <CardNews {...props} />,
-        video: <CardVideo {...props} />,
-        figure: <CardFigure {...props} />,
-    }
-    return cardTypes[type || 'base']
-}
-
+const Card: React.FC<CardProps> = ({
+    link,
+    header,
+    description,
+    image,
+}): JSX.Element => (
+    <article className='c-card' itemScope itemType='http://schema.org/Article'>
+        <a href={link} itemProp='url'>
+            <figure itemScope itemType='http://schema.org/ImageObject'>
+                <img src={image.src} alt={image.alt} loading='lazy' />
+            </figure>
+            <header>
+                <h3
+                    itemProp='name'
+                    dangerouslySetInnerHTML={{ __html: header }}
+                />
+                {description && (
+                    <p
```

Comment:
Please Create a CardDescription component to be shared to the other components

Reply:
Fixed,  added CardDescription component